src/palette/qml/MuseScore/Palette/internal/PaletteGridView.qml

```diff
@@ -463,7 +463,7 @@ StyledGridView {
         visible: false
     }
 
-    StyledMenuLoader {
+    QtObject {
```

Comment:
Is there a specific reason to keep these things here in a separate QObject, instead of putting them into the `ContextMenuLoader`?

Reply:
Only to keep the menu and the loader separated. I am fine with moving the menu into the `ContextMenuLoader` if the lack of separation is not a concern.

Reply:
Yes, please put them into ContextMenuLoader

Reply:
I've made the change amending the last commit. Thanks!